pkg/util/kubernetes/pod.go

```diff
@@ -60,8 +59,22 @@ func natsPodContainer(clusterName, version string) v1.Container {
 			},
 		},
 	}
+}
 
-	return c
+// reloaderContainer returns a NATS server pod container spec.
+func natsPodReloaderContainer() v1.Container {
+	return v1.Container{
+		Name:            "pid",
+		Image:           "wallyqs/nats-server-config-reloader:0.2.1-v1alpha2",
```

Comment:
Should the image and the pull policy be able to be overridden?

Reply:
These are now configurable:

```yaml
  pod:
    enableConfigReload: true
    reloaderImage: "connecteverything/nats-reloader"
    reloaderImageTag: "latest"
    reloaderImagePullPolicy: "Always"
```